benchmark/vllm/README.md

```diff
@@ -51,23 +51,39 @@ cat /proc/sys/kernel/numa_balancing
 
 For the experimental features and known issues concerning ROCm optimization efforts on vLLM, see the developer's guide at [ROCm/vLLM](https://github.com/ROCm/vllm/blob/main/docs/dev-docker/README.md).
 
-To change the the latency batch size, input sequence length (ISL), and output sequence length (OSL), you can modify at [vllm benchark script](../../scripts/vllm/vllm_benchmark_report.sh#L73)
+To change the the latency batch size, input sequence length (ISL), and output sequence length (OSL), you can modify at [vllm benchmark script](../../scripts/vllm/vllm_benchmark_report.sh#L96)
 
--   Default batch size: 1, 2, 4, 8, 16, 32, 64, 128, 256
--   Default ISL: 128, 2048
--   Default OSL: 1, 128
+-   Default batch size: 1 8 32 128
+-   Default ISL: 128 2048
+-   Default OSL: 128 2048
 
-Throughput input sequence length (ISL) and output sequence length (OSL) can be changed at [vllm benchark script](../../scripts/vllm/vllm_benchmark_report.sh#L78)
+Throughput input sequence length (ISL) and output sequence length (OSL) can be changed at [vllm benchmark script](../../scripts/vllm/vllm_benchmark_report.sh#L111) and other configs in the [config.csv](../../scripts/vllm/config.csv)
 *
 
--   Default ISL/OSL: 128:128, 2048:128, 128:2048, 2048:2048
+-   Default ISL: 128 2048
+-   Default OSL: 128 2048
+
+Serving number of prompts, maximum number of concurrent requests, input sequence length (ISL) and output sequence length (OSL) can be changed at [vllm benchmark script](../../scripts/vllm/vllm_benchmark_report.sh#L122)
+*
+
+-   Default prompts: 252
+-   Default concurrency: 128
+-   Default ISL: 128 2048
+-   Default OSL: 128 2048
+
+Note: The configs can also be overriden at runtime by passing in the following args:
+-   -i or --input_len
+-   -o or --output_len
+-   -b or --batch_size
+-   -p or --num_prompts
+-   -mc or --max_concurrency
 
 ### Download the Docker image 🐳
 
 The following command pulls the Docker image from Docker Hub.
 
 ```sh
-docker pull rocm/vllm:rocm6.3.1_vllm_0.8.5_20250521
+docker pull rocm/vllm-dev:nightly_0610_rc2_0610_rc2_20250605
```

Comment:
Update README to use rocm/vllm:rocm6.4.1_vllm_0.9.0.1_20250605

Reply:
Done